source/_integrations/eheimdigital.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: EHEIM Digital
+description: Instructions on how to set up EHEIM Digital with Home Assistant.
+ha_category:
+  - Light
+ha_release: 2024.11
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@autinerd'
+ha_domain: eheimdigital
+ha_integration_type: hub
+ha_platforms:
+  - light
+---
+
+The **EHEIM Digital** {% term integration %} allows you to control your [EHEIM Digital](https://eheim.com/en_GB/aquatics/eheim-digital/) smart aquarium devices from Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+## Supported devices
+
+Currently, the following devices and features are supported:
+
+### [EHEIM classicLEDcontrol+e](https://eheim.com/en_GB/aquatics/technology/lighting-control/classicledcontrol-e/classicledcontrol-e)
+
+- Controlling the brightness of both light channels
+- Enabling the daycycle mode, which automatically controls the brightness based on the daytime as configured on the device
+
+Support for additional EHEIM Digital devices will be added in future updates.
```

Comment:
```suggestion
Support for additional EHEIM Digital devices will be added in future updates.

## Entities

The EHEIM Digital integration provides the following entities:

### Lights

**Brightness**:  Allows controlling the brightness of both light channels.
**Daycycle mode**:  Automatically controls the brightness based on the daytime as configured on the device.
```
From your description, it sounds like you're adding more entities later. maybe it's worth moving this information already into a separate section.

Reply:
I have it now restructured.

Reply:
:pray: thanks!